crates/node/src/main.rs

```diff
@@ -182,6 +183,15 @@ impl workflow::TransactionStore for storage::Database {
 async fn run(config: Arc<Config>) -> Result<()> {
     let node_key = read_node_key(&config.node_key_file)?;
 
+    // Register metrics counters.
+    metrics::register_metrics();
```

Comment:
Metrics are always registered and gathered. Later on, I'll provide a separate PR that allows metrics export over P2P. The local HTTP server is optional and up to node operator whether it's enabled or not.